www/install/php/Update-21.10.0.php

```diff
@@ -18,3 +18,35 @@
  * For more information : contact@centreon.com
  *
  */
+
+
+include_once __DIR__ . "/../../class/centreonLog.class.php";
+$centreonLog = new CentreonLog();
+
+//error specific content
+$versionOfTheUpgrade = 'UPGRADE - 21.10.0: ';
```

Comment:
Shouldn't this be Update-21.10.2 ?

Reply:
Yes It should be. I've renamed the file.

Reply:
I've renamed the file